### PR TITLE
[WIP] Downtime

### PR DESCRIPTION
**Pull request by iainbeeston**
JIRA: https://jira.fundingcircle.com/browse/ENG-171

This allows users to enter the downtime for a release when they create a feature review.

### CODE REVIEW

File: app/models/repositories/impact_assessment_repository.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      @store = store
+    end
+
+    delegate :table_name, to: :store
+
+    def impact_assessments_for(versions:, at: nil)
+      query = at ? store.where('created_at <= ?', at) : store
+
+      query
+        .where('versions = ARRAY[?]::varchar[]', prepared_versions(versions))
```
Comment (andistuder) [inline app/models/repositories/impact_assessment_repository.rb]: Please make sure there are indexes that support this query

File: app/models/snapshots/impact_assessment.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require 'active_record'
+
+module Snapshots
+  class ImpactAssessment < ActiveRecord::Base
```
Comment (andistuder) [inline app/models/snapshots/impact_assessment.rb]: Have you considered adding the Impact assessment to the feature review snapshot instead?

**Comment by andistuder** [comment]
@FundingCircle/black-bat I suggest we close this PR and delete the branch. The branch has gone stale, and I reckon the changes in it don't warrant maintaining it. LMK if ok, or not.

**Comment by dideler** [comment]
Agreed. When we work on this feature we can take another look at this branch and restore it if necessary.

**Pull request closed without merging**